xgettext.js

```diff
@@ -192,13 +193,14 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 			var match = {
 				arguments: node.arguments,
 				keyword: functionName,
-				line: node.loc.start.line
+				line: node.loc.start.line,
+				column: node.loc.start.column
 			};
 
 			// Find translator comment
 			_.each( parsedInput.comments, function( translatorComment ) {
-				if ( node.loc.start.line === translatorComment.line.line ||
-					node.loc.start.line - 1 === translatorComment.line.line ) {
+				if ( node.loc.start.line === translatorComment.line ||
```

Comment:
Oh, sorry, I totally missed the changes a few lines above. The condition here reads much nicer now, though I agree since we're not using `translatorComment.column` it might be better to not include that property in the `comments.push` above.